packages/odyssey-react-mui/src/web-component/renderReactInWebComponent.ts

```diff
@@ -32,7 +32,7 @@ const SsrFriendlyHtmlElementClass =
 export class ReactInWebComponentElement extends SsrFriendlyHtmlElementClass {
   getReactComponent: GetReactComponentInWebComponent;
   reactRootElements: ReactRootElements;
-  reactRootPromise: Promise<Root>;
+  reactRootPromise: Promise<Root | null> = Promise.resolve(null);
```

Comment:
Should we add a comment somewhere about how this works? I think it'd be confusing to anyone unfamiliar with the code.

Something like: "We want to retain the order of execution for mounting and unmounting, and by using promises, that ensures the order is consistent. If we didn't have a dynamic import, we wouldn't need to chain promises like this as `createRoot.render()` and `.unmount()` are synchronous."